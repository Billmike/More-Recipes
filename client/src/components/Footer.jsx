import React from 'react';

const Footer = () => (
  <div>
    <footer class="footer">
      <div class="container">
        <span class="text-muted">Place sticky footer content here.</span>
      </div>
    </footer>
  </div>
);

export default Footer;
