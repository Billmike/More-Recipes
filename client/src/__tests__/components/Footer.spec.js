import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '../../components/Footer';

test('Should render footer correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper.find('div').length).toBe(2);
  expect(wrapper.find('span').length).toBe(1);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
