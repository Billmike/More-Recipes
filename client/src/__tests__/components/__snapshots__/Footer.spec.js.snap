// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render footer correctly 1`] = `
<div>
  <footer
    className="footer"
  >
    <div
      className="container"
    >
      <span
        className="text-muted"
      >
        Â© Michael Inc. 2018
      </span>
    </div>
  </footer>
</div>
`;
